libioc/Config/Jail/Globals.py

```diff
@@ -61,16 +64,28 @@
     "allow_sysvipc": 0,
     "allow_raw_sockets": 0,
     "allow_chflags": 0,
+    "allow_mlock": None,
     "allow_mount": 0,
     "allow_mount_devfs": 0,
+    "allow_mount_fusefs": False,
     "allow_mount_nullfs": 0,
     "allow_mount_procfs": 0,
     "allow_mount_fdescfs": 0,
     "allow_mount_zfs": 0,
     "allow_mount_tmpfs": 0,
     "allow_quotas": 0,
     "allow_socket_af": 0,
+    "allow_tun": 0,
     "allow_vmm": False,
+    "available": 0,
+    "bpf": None,
+    "comment": None,
+    "compression": None,
+    "compressratio": None,
+    "count": None,
+    "cpuset": False,
```

Comment:
There is a new feature suggested in #636 - this change should be handled in a separate PR that addresses the GitHub Issue.